S-Smans.Calculator/S-Smans.Calculator/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿using CalculatorLibrary;
+
+bool endApp = false;
+Calculator calculator = new Calculator();
+
+
+while (!endApp)
+{
+    Console.Clear();
+    Console.WriteLine("Console Calculator in C#\r");
+    Console.WriteLine("------------------------");
+    Console.WriteLine($"    {calculator.Iterations} calculation\n");
+
+    string numInput1 = "";
+    string numInput2 = "";
+    double result = 0;
+
+    Console.Write("Type a number, and then press Enter: ");
+    numInput1 = Console.ReadLine();
+
+    double cleanNum1 = 0;
+    while (!double.TryParse(numInput1, out cleanNum1))
+    {
+        Console.Write("This is not valid input. Please enter an integer value: ");
+        numInput1 = Console.ReadLine();
+    }
+
+    Console.Write("Type another number, and then press Enter: ");
+    numInput2 = Console.ReadLine();
+
+    double cleanNum2 = 0;
+    while (!double.TryParse(numInput2, out cleanNum2))
+    {
+        Console.Write("This is not valid input. Please enter an integer value: ");
```

Comment:
Here we need to tell the user that split numbers are allowed, not just integers.

Reply:
renamed to numeric.